src/aind_metadata_mapper/stimulus/camstim.py

```diff
@@ -84,6 +89,29 @@ def __init__(
         if self.opto_table_path.exists():
             self.stim_epochs.append(self.epoch_from_opto_table())
 
+    def get_folder(self, session_id) -> str:
+        """returns the directory name of the session on the np-exp directory"""
+        for subfolder in NPEXP_ROOT.iterdir():
+            if subfolder.name.split("_")[0] == session_id:
+                return subfolder.name
+        else:
+            raise Exception("Session folder not found in np-exp")
+
+    def get_npexp_path(self, session_id) -> Path:
+        """returns the path to the session on allen's np-exp directory"""
+        return NPEXP_ROOT / self.get_folder(session_id)
+
+    def get_session_uuid(self) -> str:
+        """returns session uuid from pickle file"""
+        return pkl.load_pkl(self.pkl_path)["session_uuid"]
+
+    def get_mtrain(self) -> dict:
+        """Returns dictionary containing 'id', 'name', 'stages', 'states'"""
+        server = "http://mtrain:5000"
```

Comment:
Same comment as above: It's not recommended to hard-code endpoints. It'll be better to make this configurable.